.github/workflows/ci.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
 
       - name: Check changed files
         id: check_changed_files
+        continue-on-error: true
         run: |
           git diff --name-only --exit-code
```

Comment:
I don't know how much we care about consistency, but connect-playwright does a diff check like [this](https://github.com/connectrpc/connect-playwright-es/blob/main/.github/workflows/ci.yaml#L32-L35). Doesn't matter to me which one we use, but we should probs settle on one.

Reply:
I'm all up for a consistent one. I find this very readable without bash gymnastics but my bash skills are also pretty weak so maybe it's just me.

Reply:
Haha, yeah, agreed. But, I don't think `git diff` alerts on untracked files?

```bash
$ touch checkdifftest                                                                                                                                                                                                                                                                                                                                                             
$ git status

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        checkdifftest

nothing added to commit but untracked files present (use "git add" to track)

$ git status --porcelain                                                                                                                                                                                                                                                                                                                                                          
?? checkdifftest

$ git diff --name-only --exit-code
# no output
```

Reply:
oh yeah, good point